tests/testenv.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+usage()
+{
+	echo "$0 [arch: x86/x64/arm/arm64] [flavor: debug/release]"
+	exit 1
```

Comment:
Currently, when we run coreclr tests, the runtest bash/batch script sets up all the variables needed for ilc.cmd script which will hook the toolchain to compile the coreclr test.

When we are debugging and want to just run a single test. Using the testenv and restore scripts will setup these variables, so for example, Add1.cmd  can be run directly without the whole runtest.cmd.

This is mainly for communication when coreclr xunit runner invokes:
%CLRCustomTestLauncher% %~dp0 test.exe %...Args%

The Custom Test Launcher is ilc.cmd which uses pre-set environment variables.